microsoftgraph/msgraph-sdk-python-core
# Fix bug on prepared request in graph client

## Overview

Fixes a bug on the GraphClient when creating a prepared request. 

![image](https://user-images.githubusercontent.com/40166690/125309938-d240c300-e33a-11eb-8054-ed53928861f2.png)



### Notes

The error would occur as a result of passing `**kwargs` when instantiating a `Request` object as opposed to passing only the expected keyword arguments.

## Testing Instructions

* Checkout this branch
* Instantiate a client using the GraphClient
* Make a request to the graph and pass request specific kwargs such as `stream=True` 
* Observe that the request works as expected

Fixes #98
Fixes #99
